dot/state/offline_pruner.go

```diff
@@ -131,14 +131,14 @@ func (p *OfflinePruner) SetBloomFilter() (err error) {
 		blockNum = header.Number
 	}
 
-	for key := range merkleValues {
-		err = p.bloom.put([]byte(key))
+	for key := range nodeHashes {
+		err = p.bloom.put(key.ToBytes())
 		if err != nil {
 			return err
 		}
 	}
 
-	logger.Infof("Total keys added in bloom filter: %d", len(merkleValues))
+	logger.Infof("Total keys added in bloom filter: %d", len(nodeHashes))
```

Comment:
are we replacing the bloom filter? Should we change this log message to be more general?

```suggestion
	logger.Infof("Total keys added in prune filter: %d", len(nodeHashes))
```

Reply:
Changed the log line completely (should had done it in #3084 my bad)